README.rst

```diff
@@ -15,7 +15,7 @@ Django Mail Factory lets you manage your email in a multilingual project.
 * Authors: Rémy Hubscher and `contributors
   <https://github.com/novafloss/django-mail-factory/graphs/contributors>`_
 * Licence: BSD
-* Compatibility: Django 1.8, 1.9, 1.10, python2.7 and python3.4
+* Compatibility: Django 1.8, 1.9, 1.10, 1.11, python2.7, python3.4 and python3.5
```

Comment:
Consider adding Python 3.6 there as well :)

Reply:
I'll make a new release right after this one. 😉 
Do you think we can drop python3.4 in the same time @Natim?

Reply:
To be honest, people using Python 3 are really likely to upgrade to the last Python version so I wouldn't mind dropping it and keeping the two last versions